fuzz/src/bin/gen_target.sh

```diff
@@ -14,29 +14,26 @@ GEN_TEST router
 
 GEN_TEST msg_accept_channel msg_targets::
```

Comment:
Why is a .sh file being formatted?

Reply:
I had to edit some of these files so the autogenerated code matches what Rustfmt produces. Alternately, I could have these .sh files add skips to the files they generate.

Making rustfmt ignore a file is an unstable feature, sadly.

Reply:
Right, but why reorder things within the file? That shouldnt effect the generated formats?

Reply:
ah true, fixed

Reply:
Next question, why is a .h file being formatted by rustfmt?

Reply:
Uh that's a good point. Updated!